test/test_view_drawing.py

```diff
@@ -95,3 +97,21 @@ def test_change_transparency_slider_value(qtbot, test_object, init_config):
 
     # Run Drawing reset, prevents post test crash
     test_object.draw_window.onResetClicked()
+
+
+def test_manual_drawing(qtbot, test_object, init_config):
+    """Function to create a drawing, press fill, press draw, and assert that the drawing is in draw "mode"."""
+    # Assert Drawing window exists
+    test_object.draw_window.show()
+    assert test_object.draw_window.windowTitle() == "OnkoDICOM - Draw Region Of Interest"
+
+    # Assert drawing is in draw "mode"
+    test_object.draw_window.min_pixel_density_line_edit.setText("900")
+    test_object.draw_window.max_pixel_density_line_edit.setText("1000")
+    qtbot.mouseClick(test_object.draw_window.image_slice_number_fill_button, QtCore.Qt.LeftButton)
+    qtbot.mouseClick(test_object.draw_window.image_slice_number_draw_button, QtCore.Qt.LeftButton)
+    assert test_object.draw_window.keep_empty_pixel is not False
+    assert test_object.draw_window.drawingROI.is_drawing is not False
```

Comment:
pretty sure you can simplifiy this to assert X (as it will be true)

Reply:
Resolved by changing to is True